packages/features/ee/teams/components/CreateANewTeamForm.tsx

```diff
@@ -42,8 +43,13 @@ export const CreateANewTeamForm = () => {
 
   const createTeamMutation = trpc.viewer.teams.create.useMutation({
     onSuccess: (data) => {
-      telemetry.event(telemetryEventTypes.team_created);
-      router.push(`/settings/teams/${data.id}/onboard-members`);
+      telemetry.event(telemetryEventTypes.team_checkout_session_created);
```

Comment:
Since a team isn't really being created at this stage let's create a new event. That way, we can see if people are abandoning the process.